<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormShaker</class>
 <widget class="QMainWindow" name="FormShaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Booster Shaker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
  <widget class="QGroupBox" name="titleGroupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>710</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="titleGroupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>710</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Shaker Amplifiers</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 1</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:getVoltage</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:getCurrent</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:getAmplifierTemperature</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:getBoardTemperature</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:getRFPower</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDetails1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QESimpleShape" name="qesimpleshape_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>22</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:isEnabled</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDetails2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH2:getCurrent</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH2:getBoardTemperature</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH2:getRFPower</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH2:getAmplifierTemperature</string>
    </property>
   </widget>
   <widget class="QESimpleShape" name="qesimpleshape_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>42</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH2:isEnabled</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH2:getVoltage</string>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledInterlock1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>22</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Warning</enum>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledInterlock2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>42</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Warning</enum>
    </property>
   </widget>
   <widget class="QEPushButton" name="qepushbutton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:enable</string>
    </property>
    <property name="clickText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QEPushButton" name="qepushbutton_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:enable</string>
    </property>
   </widget>
   <widget class="QEPushButton" name="qepushbutton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH2:enable</string>
    </property>
    <property name="clickText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QEPushButton" name="qepushbutton_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
    <property name="variable" stdset="0">
     <string>BO-DI-SHK1-CH1:enable</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Libera Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Shaker Mode</string>
   </property>
  </widget>
  <widget class="QEComboBox" name="qecombobox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>190</width>
     <height>23</height>
    </rect>
   </property>
   <property name="variable" stdset="0">
    <string>BO-DI-SHK1:setOperationMode</string>
   </property>
  </widget>
  <widget class="QEPushButton" name="qepushbutton_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
   <property name="variable" stdset="0">
    <string>BO-DI-SHK1:setLiberaShakerMode</string>
   </property>
   <property name="clickText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QEPushButton" name="qepushbutton_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
   <property name="variable" stdset="0">
    <string>BO-DI-SHK1:setLiberaShakerMode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplifier Temp.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Board Temp.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RF Power</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Interlock</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEComboBox</class>
   <extends>QComboBox</extends>
   <header>QEComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
